lib/sidekiq_publisher/client.rb

```diff
@@ -4,29 +4,77 @@
 
 module SidekiqPublisher
   class Client < Sidekiq::Client
+    # A client that can be used to bulk-publish jobs of many different types together in
+    # a single atomic push. Unfortunately, Sidekiq does not provide an API to do this --
+    # their bulk API only supports an array of args for a single job class.
```

Comment:
This comment is mostly taken from [the original PR which added this class](https://github.com/ezcater/sidekiq_publisher/pull/5).